ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/project.pbxproj

```diff
@@ -616,6 +620,7 @@
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = 9KBH5RKYEW;
 				INFOPLIST_FILE = FluentUI.Demo/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
podspec needs to change too

Reply:
While we're touching this seems like a good chance to pull out at least the setting we're touching into an xcconfig instead of in the project.pbxproj itself to avoid having to duplicate the settings and to eventually make merges easier as the project grows

Reply:
This was addressed in #320 for the master branch and the modifications made in the vnext-prototype as part of #325.